### ISSUE TITLE
Roadmap?

### ISSUE THREAD

**Issue by transat** [Unity-Technologies/VolumetricLighting#29]
Is this project still being actively developed? And if so, is there a roadmap for when things like directional lights will be working?

**Comment by rsodre** [comment]
Yes, please! This was published 2 years ago, I bet this code would not even work on current Unity versions. By now I would expect we could just turn it on in a project, that's so frustrating.

**Comment by markmorris** [comment]
From the replies im assuming its no longer being developed which is a shame.

**Comment by builder-main** [comment]
It won't evolve as HDRP will include its own fog system (https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@6.9/manual/Override-Volumetric-Fog.html , check version link).
However I confirm this old VolumetriLightning effect works with unity 2019.3 beta
